README.md

```diff
@@ -141,7 +141,7 @@ Here we have specified `number` for each parameter type, and `boolean` for the r
 
 ### Type Declarations
 
-Much of the time, developers need to create new names for types so that they can be easily referenced without repetition and so that they can be declared recursively.
+Most of the time, developers need to create new names for types so that they can be easily referenced without repetition and so that they can be declared recursively.
```

Comment:
Heh, "much of the time" is a @DanielRosenwasser -ism and it's neither incorrect grammar nor a typo. Should be fine to keep as-is I think 👍🏻 

```suggestion
Much of the time, developers need to create new names for types so that they can be easily referenced without repetition and so that they can be declared recursively.
```